cfgov/v1/atomic_elements/organisms.py

```diff
@@ -560,3 +561,26 @@ class VideoPlayer(blocks.StructBlock):
     class Meta:
         icon = 'media'
         template = '_includes/organisms/video-player.html'
+
+
+class HTMLBlock(blocks.StructBlock):
+    html_url = blocks.RegexBlock(
+        label='Source URL',
+        default='',
+        required=True,
+        regex=r'^https://s3.amazonaws.com/files.consumerfinance.gov/.+$',
```

Comment:
Can we make the S3 part optional so that we can seamlessly transition to `https://files.consumerfinance.gov` in the future? 

I'm thinking: `^https://(s3.amazonaws.com/)?files.consumerfinance.gov/.+$`